pyaedt/sbrplus/matlab/draw_rays1.m

```diff
@@ -16,14 +16,15 @@
 %                    controls color for ray lines, hit points, and footprints,
 %                    unless overridden by .colorByField
 %
-%   .colorByFld (str) {'Einc','Hrefl','Jtot','Mtrans', etc.} activates
-%                     coloring ray footprints and hit points according
-%                     do E or H field quantities, in general any combination
+%   .colorByFld (str) {'Einc','Hrefl','Jtot','Mtrans', etc., or 'dist'}
```

Comment:
@robertkipp Can you please add a space after a comma?

Reply:
yes